habitat-lab/habitat/datasets/rearrange/README.md

```diff
@@ -0,0 +1,15 @@
+## Generating episodes for Object Rearrangement task
+
+Given the standard GeoGoal rearrange episodes data at <data_dir>/<split>/<geogoal_episodes_name>.yaml, the script `convert_to_object_rearrange_episodes.py` can be used to generate the corresponding episodes for Object Rearrangement task. The command for using this script is as follows:
+
+```
+python habitat-lab/habitat/datasets/rearrange/convert_to_object_rearrange_episodes.py --data_dir <data_dir> --source_episodes_tag <geogoal_episodes_name> --target_episodes_tag <target_episodes_name> --obj_category_mapping_file /path/to/csv/with/object_category_mapping --rec_category_mapping_file /path/to/csv/with/receptacle_category_mapping
```

Comment:
Where is the `convert_to_object_rearrange_episodes` file?

Reply:
Sorry, I renamed it. [habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py](https://github.com/facebookresearch/habitat-lab/pull/1050/files/0b08919c3951c0c030a4a94ed7fd342190201279#diff-aed663b40b1037a3932fbca619c2911cd788fcfef320f00835e5533bf8b9371d). I will update this.